File: managment_tournoi_backend/src/main/java/ma/youcode/managment_tournoi_backend/dto/appUserFileDto/getDto/MemberShowDto.java
package ma.youcode.managment_tournoi_backend.dto.appUserFileDto.getDto;

import lombok.Builder;
import lombok.Data;
import ma.youcode.managment_tournoi_backend.entity.AppRole;

import java.time.LocalDateTime;
import java.util.UUID;
@Data
@Builder
public class MemberShowDto {
    private UUID id;
    private String firstName;
    private String lastName;
    private String username;
    private String email;
    private String urlPicture;
    private String className;
    private LocalDateTime createdAt;
    private AppRole role;
}

File: managment_tournoi_frontend/src/app/dto/appUserFileDto/getDto/memberShowDto.ts
import { AppRole } from "../../roleDto/AppRole";

export interface MemberShowDto {
    id: string; 
    firstName: string;
    lastName: string;
    username: string;
    email: string;
    urlPicture: string;
    className: string;
    createdAt: string;
    role: AppRole;
}
